### PR TITLE
Add gitpod button on README and create .gitpod.yml and add deploy script

### LINKED ISSUE #2: [FEATURE] Add gitpod preview
### Detailed description

Add gitpod to quick contribute into this project

### Context

Add gitpod to quick contribute into this project

### Possible implementation

_No response_

### Additional information

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by hyomiTheTech**
## Related Issue

- Information about the related issue

Closes: # 2

### Describe the changes you've made

1. Added Gitpod button on README.md
2. Created a simple .gitpod.yml file with a basic "init" and "deploy" script.

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update.
- [ ] This change requires a documentation update

## How Has This Been Tested?

I checked out my branch and checked the Gitpod button working. But I have not tested the "deploy" script working.

## Checklist

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented on my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots (if applicable)

 Original           | Updated
 :--------------------: |:--------------------:
 original screenshot | updated screenshot |
 
## Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/web3community/.github/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: .gitpod.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# This configuration file was automatically generated by Gitpod.
+# Please adjust to your needs (see https://www.gitpod.io/docs/config-gitpod-file)
+# and commit this file to your remote git repository to share the goodness with others.
+
+tasks:
+  - init: npm install && npm run build
+    command: npm run start
+  - deploy: next build
```
Comment (hyomiTheTech) [inline .gitpod.yml:8]: This whole file is a basic initial .gitpod.yml, and I added line #8 ~ 9. Please check my code. Thanks!

**Comment by hyomiTheTech** [comment]
Thanks for allowing me to make my very first commit to the community! I am super excited and ready to contribute further : )

File: .gitpod.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    command: next build
+vscode:
+  extensions:
+    - dbaeumer.vscode-eslint
```
Comment (shuklaritvik06) [inline .gitpod.yml:12]:
```suggestion
    - dbaeumer.vscode-eslint
    
```

**Review by mkubdev — APPROVED** [review]
Hello @hyomiTheTech ! Thanks for your contribution, all look good to me 🚀
You can join us on [our discord](https://discord.gg/TSRwqx4K2v).

**Comment by hyomiTheTech** [comment]
> Hello @hyomiTheTech ! Thanks for your contribution, all look good to me 🚀 You can join us on [our discord](https://discord.gg/TSRwqx4K2v).

I already did!

**Pull request merged**